stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -183,7 +183,9 @@ spec:
           cp -fv /tmp/migrationHelmInfo.yaml $scriptsPath/migrationHelmInfo.yaml;
           cp -fv /tmp/migrationStatus.sh $scriptsPath/migrationStatus.sh;
           mkdir -p {{ .Values.artifactory.persistence.mountPath }}/log;
-          bash $scriptsPath/migrationStatus.sh {{ .Chart.AppVersion }} > >(tee {{ .Values.artifactory.persistence.mountPath }}/log/helm-migration.log) 2>&1;
+          bash $scriptsPath/migrationStatus.sh {{ default .Chart.AppVersion .Values.artifactory.image.version }} > >(tee {{ .Values.artifactory.persistence.mountPath }}/log/helm-migration.log) 2>&1;
+        resources:
+{{ toYaml .Values.initContainers.resources | indent 10 }}
```

Comment:
The migration container will not work with `initContainers.resources` as it actually starts Artifactory. Need to provide it with the same resources as the node is set: `artifactory.node.resources`.

Reply:
Sweet as. I'll update the primary migration container as well